hackvote/config/routes.rb

```diff
@@ -1,4 +1,13 @@
 Rails.application.routes.draw do
+  root        'events#new'
+
+  post 'vote' => 'projects#vote'
+  post 'close' => 'events#close'
```

Comment:
These should be under the resources. See http://guides.rubyonrails.org/routing.html#adding-more-restful-actions